### DISCUSSION TITLE
Write to debug

### DISCUSSION THREAD

**Discussion by jajabu33** [zero-plusplus/vscode-autohotkey-debug#216] [Q&A] [answered]
When I run a script using vscode and try this code
```ahk
FileAppend,  Test`n,*
```
It prints to the console `test`, i'm not sure if it's possible through the extension, but I would like to ask 
if you know a way to print to `vscode debug console` from a script that is not running through it

I mean for example, when you run the script by double-clicking on it, instead of opening it on vscode

**Comment by zero-plusplus** [comment, marked as answer]
In conclusion, it is not possible.

I tried to find a way to get the contents of the standard output of an already running process a long time ago, but came to the conclusion that it is not possible.

The reason this extension can output messages to standard output is because it gets a handle on standard output by running an AutoHotkey process within the extension.

---

As a suggestion, if you want to get messages from an already running AutoHotkey script, you can use [OutputDebug](https://www.autohotkey.com/docs/commands/OutputDebug.htm).
The messages can be viewed by using Microsoft's [DebugView](https://docs.microsoft.com/en-us/sysinternals/downloads/debugview) software.

Also, this extension supports OutputDebug, which is superior to FileAppend if you want to output messages.
